contrib/completion/bash/docker

```diff
@@ -169,6 +169,22 @@ __docker_capabilities() {
 	" -- "$cur" ) )
 }
 
+# a selection of the available signals that is most likely of interest in the
+# context of docker containers.
+__docker_signals() {
+	COMPREPLY=( $( compgen -W "
+		CONT
+		HUP
+		INT
+		KILL
+		QUIT
+		STOP
+		TERM
+		USR1
+		USR2
```

Comment:
These can also legally be specified with an optional `SIG` prefix -- can we tab complete those too?  This particular list looks solid to me -- I can't think of any others I'd want/need regularly.